packages/web_benchmarks/lib/src/recorder.dart

```diff
@@ -247,22 +247,23 @@ abstract class SceneBuilderRecorder extends Recorder {
         rethrow;
       }
     };
-    window.onDrawFrame = () {
+    PlatformDispatcher.instance.onDrawFrame = () {
       try {
         _profile.record('drawFrameDuration', () {
           final SceneBuilder sceneBuilder = SceneBuilder();
           onDrawFrame(sceneBuilder);
           _profile.record('sceneBuildDuration', () {
             final Scene scene = sceneBuilder.build();
             _profile.record('windowRenderDuration', () {
-              window.render(scene);
+              // TODO(goderbauer): Migrate to PlatformDispatcher.implicitView once v3.9.0 is the oldest supported Flutter version.
+              window.render(scene); // ignore: deprecated_member_use
```

Comment:
The replacement API for this particular call was only introduced recently, so we cannot migrate to it yet to remain backwards compatible with older versions. Ignoring the deprecation warning for the upcoming deprecation in https://github.com/flutter/engine/pull/39302.